frontend/src/pages/docs/[...slug].tsx

```diff
@@ -373,7 +373,12 @@ const DocContent = styled.div`
   }
 `;
 
-const StyledImage = styled(Image)();
+const StyledImage = styled(Image)`
```

Comment:
Oh, this is great! You no longer need to address the Image styles in the ImageContainer. Brilliant! 😯